bee-ledger/src/workers/pruning/condition.rs

```diff
@@ -1,77 +1,50 @@
-// Copyright 2020-2021 IOTA Stiftung
+// Copyright 2020 IOTA Stiftung
```

Comment:
Seems like multiple Copyright headers are not consistent, please check the files.

Reply:
@thibault-martinez what's the way/policy here?